authenticate/handlers/webauthn/webauthn.go

```diff
@@ -302,7 +302,17 @@ func (h *Handler) handleRegister(w http.ResponseWriter, r *http.Request, state *
 			Id: webauthnutil.GetDeviceCredentialID(serverCredential.ID),
 		},
 	})
-	return h.saveSessionAndRedirect(w, r, state, redirectURIParam)
+
+	// add the device credential ID to the redirect uri
+	redirectURI, err := urlutil.ParseAndValidateURL(redirectURIParam)
+	if err != nil {
+		return err
+	}
+	q := redirectURI.Query()
+	q.Set(urlutil.QueryDeviceCredentialID, deviceCredentialID)
+	redirectURI.RawQuery = q.Encode()
+
+	return h.saveSessionAndRedirect(w, r, state, redirectURI.String())
```

Comment:
I believe this creates a signed url right?

Reply:
I did not change the behavior of this.